frontend/components/TargetLabelSelector/_styles.scss

```diff
@@ -10,19 +10,6 @@
     margin: $pad-medium 0;
   }
 
-  &__no-labels {
-    display: flex;
-    height: 187px;
```

Comment:
do we need a specific height set here?

Reply:
No, the `"form"` class of the parents take care of it with `gap`